cmd/compose/up.go

```diff
@@ -181,6 +186,9 @@ func runUp(
 	if err != nil {
 		return err
 	}
+	if !upOptions.navigationMenuChanged {
```

Comment:
seems bit complicated to manage value set by env var, is there any drawback doing the same as https://github.com/docker/compose/blob/main/cmd/compose/up.go#L109-L110 ?

Reply:
(this comment is mine, I was connected with by @docker.com email while commenting :P)

Reply:
we also want to have into consideration the value of the Docker Desktop experimental feature value 
<img width="624" alt="Screenshot 2024-03-21 at 16 05 49" src="https://github.com/docker/compose/assets/22371565/76052e99-8dd2-442b-85f6-6643f1865c6c">

Reply:
oh indeed